solidity/deploy/05_deploy_bridge.ts

```diff
@@ -31,32 +31,36 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
   })
 
-  const Bridge = await deploy("Bridge", {
-    contract:
+  const bridge = await helpers.upgrades.deployProxy("Bridge", {
```

Comment:
Do we need to workaround the gas reporter bug as in https://github.com/keep-network/keep-core/pull/2970?

Reply:
Actually, the reporter works here. I created https://github.com/keep-network/keep-core/issues/2979 to investigate it further.